tests/unit/test_view_model.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from todo_app.src.trello import Item
+from todo_app.src.view import ViewModel
+
+
+MOCK_ITEMS = [
+    Item(id="id_1", name="1", status="To do"),
+    Item(id="id_2", name="2", status="Done"),
+]
+
+
+@pytest.fixture(name="default_mock_view")
+def get_default_mock_view():
+    return ViewModel(items=MOCK_ITEMS)
+
+
+def test_get_items(default_mock_view):
+    assert default_mock_view.items == MOCK_ITEMS
+
+
+def test_get_done_items(default_mock_view):
+    assert all([item.status == "Done" for item in default_mock_view.done_items])
```

Comment:
I'd always be tempted to pair this with a check of the count - there's some very viable bugs that could let this check pass; e.g. if we're accidentally filtering on the wrong thing such that _no items get selected_, then triviailly all zero of them are "Done".
```suggestion
    assert all([item.status == "Done" for item in default_mock_view.done_items])
    assert len(default_mock_view.done_items) == 1
```

Reply:
(Can check this by switching your view model to just `return []` in the done property)

Reply:
(Also, note this is me assuming that this would return a list, but that's not actually currently true - see below)

Reply:
Good shout